UPGRADE-4.4.md

```diff
@@ -72,6 +72,8 @@ Filesystem
 Form
 ----
 
+ * Using different values for the "model_timezone" and "view_timezone" options without configuring a reference date is
```

Comment:
you should say `the "model_timezone" and "view_timezone" options of the TimeType`. These options also exist for other types (DateType and DatetimeType) for which there is no such deprecation AFAICT.